src/main.cpp

```diff
@@ -290,6 +292,19 @@ void process_file(padded_string_view input, error_reporter *error_reporter,
 }
 
 void print_help_message() {
+  const int max_width = 36;
+
+  auto print_helper = [&](const char *abbr, const char *message) {
+    std::cout << std::setw(max_width) << std::left << abbr << message << std::endl;
+  };
+
+  std::cout << "Usage: quick-lint-js [OPTIONS]... [FILE]..."
+            << std::endl << std::endl
```

Comment:
Prefer `"\n`" over `std::endl`. It's shorter and faster (not that speed matters much here...).